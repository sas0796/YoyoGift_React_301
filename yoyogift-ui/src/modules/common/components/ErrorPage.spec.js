import React from "react";
import { mount } from "enzyme";

import ErrorPage from "./ErrorPage";

describe("Error Page Test Suite", () => {
  it("should render expected HTML", () => {
      expect(mount(<ErrorPage />).html()).toMatchSnapshot();
  });
});