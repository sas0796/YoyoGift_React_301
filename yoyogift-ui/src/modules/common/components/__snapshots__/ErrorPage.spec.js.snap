// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Error Page Test Suite should render expected HTML 1`] = `"<img src=\\"404.png\\" alt=\\"404: Page not found\\" style=\\"width: 100%; height: 100vh;\\">"`;
